baybe/constraints/discrete.py

```diff
@@ -129,6 +129,16 @@ def get_invalid(self, data: pd.DataFrame) -> pd.Index:  # noqa: D102
 
         return data.index[mask_bad]
 
+    def to_polars(self) -> pl.Expr:  # noqa: D102
```

Comment:
Not related to this PR (so you can ignore: @rjavadi):

@Scienfitz: Before we merge everything into main, I think we should clean up the constraint logic so that also our pandas versions return the "valid" entries – I think it's rather confusing that the two implementations return the opposite content

Reply:
can do, please remember for the merge PR of `dev/polars` into `main`